src/main/java/com/ryanharter/autoparcel/AutoParcelException.java

```diff
@@ -0,0 +1,7 @@
+package com.ryanharter.autoparcel;
+
+/**
+ * Created by rharter on 8/12/15.
+ */
+public class AutoParcelException extends RuntimeException {
```

Comment:
doesn't need to be public since it seems you only use it for busting out of the processor